README.md

```diff
@@ -6,6 +6,11 @@ This project replaces the legacy "VIVO Installer". It is used to create a "third
 
 1. Clone this repository
 2. Add any overlay files to their expected location with the `src` directory tree
+   e.g.
+   ```
+   src/main/webapp/themes/custom-theme
+   src/main/java/com/wok/vivo/dataservice/MyCustomClass.java
```

Comment:
Would it be helpful to create empty directories for:
- `src/main/webapp/themes` and
- `src/main/java/` ?

..similar to: 
- https://github.com/vivo-community/vivo-custom-webapp/tree/main/home/src/main/resources/config and
- https://github.com/vivo-community/vivo-custom-webapp/tree/main/home/src/main/resources/rdf

Reply:
Sure, take a look at what I added.